pika/connection.py

```diff
@@ -53,7 +56,7 @@ class Parameters(object):
     :param str DEFAULT_VIRTUAL_HOST: '/'
     :param str DEFAULT_USERNAME: 'guest'
     :param str DEFAULT_PASSWORD: 'guest'
-    :param int DEFAULT_HEARTBEAT_INTERVAL: None
+    :param int DEFAULT_HEARTBEAT_TIMEOUT: None
```

Comment:
RabbitMQ actually interprets the heartbeat value that we pass it in Connection.Tune as a timeout rather than an interval between heartbeats. RabbitMQ will send several Heartbeat frames during each heartbeat-timeout period.